app/controllers/starter_code_groups_controller.rb

```diff
@@ -0,0 +1,111 @@
+# Controller for starter code groups
+class StarterCodeGroupsController < ApplicationController
+  before_action { authorize! }
+
+  respond_to :js
+
+  def create
+    StarterCodeGroup.create(create_params)
```

Comment:
It would be nice to query for an assignment first (symmetric with the other methods) and then do a `assignment.starter_code_groups.create`.